src/services/escola-lms/products.ts

```diff
@@ -7,18 +7,16 @@ export async function products(
     API.PaginationParams &
     EscolaLms.Cart.Http.Requests.ProductSearchRequest & {
       type?: 'single' | 'bundle';
-      purchasable?: boolean | 0 | 1;
-      free?: boolean | 0 | 1;
+      purchasable?: 0 | 1;
+      free?: boolean | number;
```

Comment:
```suggestion
```
those 2 value are `boolean` like defined in `EscolaLms.Cart.Http.Requests.ProductSearchRequest`